### DISCUSSION TITLE
Is it possible to generate the same mock in multiple directories?

### DISCUSSION THREAD

**Discussion by gabrielgasp** [vektra/mockery#793] [Q&A] [answered]
When using mockery.yaml, we can specify the output directory of an interface mock by setting config.dir to the desired path.

I'm trying to find a way to output said mock to multiple directories in my application. Tried using a list of paths in the config.dir but without success.

Is that even possible?

**Comment by LandonTClipp** [comment, marked as answer]
Yes! I do believe this is possible. https://vektra.github.io/mockery/latest/features/#examples

There is a `configs` section you can specify (contrasted to `config`) that takes a list of config mappings. If you specify a different directory for each instance of the output mock, it should work.

**Reply by Chapalax** [reply]
Do you have a new link? Right now it returns 404 :(

**Reply by LandonTClipp** [reply]
https://vektra.github.io/mockery/v3.6/configuration/

View the extended example. Also your can view the entire config structure here: https://pkg.go.dev/github.com/vektra/mockery/v3/config#RootConfig

**Comment by gabrielgasp** [comment]
> Yes! I do believe this is possible. https://vektra.github.io/mockery/latest/features/#examples
> 
> 
> 
> There is a `configs` section you can specify (contrasted to `config`) that takes a list of config mappings. If you specify a different directory for each instance of the output mock, it should work.

Tried and it works!

Thanks a lot for the quick response.
